### ISSUE TITLE
Tree-sitter for highlighting code in cli

### ISSUE THREAD

**Issue by BonaBeavis** [tree-sitter/tree-sitter#747]
Is it possible to use tree-sitter as an highlighter in a terminal, like [highlight](http://www.andre-simon.de/doku/highlight/en/highlight.php)?

**Comment by patrickt** [comment]
Yes; the `tree-sitter highlight` command should do what you want, or serve as a good jumping-off point for what you need to do.

**Comment by BonaBeavis** [comment]
Yes, I looked into this, but this requires to clone each parser repo, or is there a metarepo with all the parsers, package.jsons and query files?

**Comment by Razzeee** [comment]
There is this, but it's not very up to date https://github.com/Menci/tree-sitter-wasm-prebuilt
